app/controllers/api_controller.rb

```diff
@@ -18,9 +17,21 @@ def tag_gif
   end
 
   def delete_gif
+    gif = UserGif.find(params[:id]) rescue nil
```

Comment:
Instead of `rescue nil` you can do `UserGif.find_by(id: params[:id])` and it will return `nil` if there is no `UserGif` with that ID.